cfme/tests/v2v/test_post_migrations.py

```diff
@@ -1,191 +1,94 @@
 from cfme.utils.appliance.implementations.ui import navigate_to
-from cfme.utils.appliance.implementations.ui import navigator
-from cfme.utils.log import logger
 from cfme.utils.wait import TimedOutError
-from cfme.utils.wait import wait_for
-
 
 pytestmark = [
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
+        required_flags=["v2v"],
         scope="module"
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
+        required_flags=["v2v"],
+        fixture_name="source_provider",
         scope="module"
-    )
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup")
 ]
 
 
-def get_migrated_vm_obj(src_vm_obj, target_provider):
-    """Returns the migrated_vm obj from target_provider."""
-    collection = target_provider.appliance.provider_based_collection(target_provider)
-    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
-    return migrated_vm
-
-
-@pytest.mark.parametrize(
-    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
-)
-def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
-                              form_data_vm_obj_single_datastore, soft_assert):
-    """Test policy to prevent source VM from starting if migration is complete
+@pytest.mark.tier(1)
+def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini, soft_assert):
+    """
+    Test to validate v2v post-migrations usecases
 
     Polarion:
         assignee: ytale
         initialEstimate: 1/4h
+        caseimportance: critical
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
         casecomponent: V2V
     """
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
-
-    @request.addfinalizer
-    def _cleanup():
-        infrastructure_mapping_collection.delete(mapping)
-
-    migration_plan_collection = appliance.collections.v2v_plans
-    # vm_obj is a list, with only 1 VM object, hence [0]
-    vm_obj = form_data_vm_obj_single_datastore.vm_list[0]
+    src_vm_obj = mapping_data_vm_obj_mini.vm_list[0]
+    source_view = navigate_to(src_vm_obj, "Details")
+    summary = source_view.entities.summary("Properties").get_text_of("Container")
+    source_cpu, source_socket, source_core, source_memory = re.findall("\d+", summary)
+    migration_plan_collection = appliance.collections.v2v_migration_plans
     migration_plan = migration_plan_collection.create(
         name="plan_{}".format(fauxfactory.gen_alphanumeric()),
         description="desc_{}".format(fauxfactory.gen_alphanumeric()),
```

Comment:
We also need to provide target provider now for OSP to run.
https://github.com/ManageIQ/integration_tests/pull/8694/files#diff-12466afeba20d2a418487020feaecf7cR151

Reply:
ahh, got you. thanks for searching this. :)

But I still wonder why we are externally adding target_provider in plan since we are not using that anywhere. I have an idea, let me see.

Reply:
ahh, got you. thanks for searching this. :)